packages.txt

```diff
@@ -0,0 +1 @@
+git make ccache cpio curl gnupg universal-ctags cscope socat patch gperf quilt byacc python3 python3-pip python3-virtualenv device-tree-compiler tmux lsb-release qemu-system-aarch64 qemu-system-riscv64
```

Comment:
Do we need to install `qemu-system-*` at all?

Reply:
No. We can get rid of it it is installed we use custom built packages.